package.json

```diff
@@ -76,26 +85,30 @@
     "jscodeshift": "0.6.4",
     "playroom": "0.15.1",
     "raw.macro": "0.3.0",
-    "react-dom": "16.11.0",
+    "react-dom": "^16.10.2",
```

Comment:
Was the version downgrade here intentional?

Reply:
I originally removed `react-dom` as we don't use it in PC at all, and listed it only as a peerDep instead, but the `reach` deps complained. When I added it back I just made it match the version we specify for React.